tutorial/creating_packages/add_dependencies_to_packages.rst

```diff
@@ -1,2 +1,133 @@
+.. _creating_packages_add_dependencies_to_packages:
+
 Add dependencies to packages
 ============================
+
+.. important::
+
+    In this example, we will retrieve the CMake Conan package from a Conan repository with
+    packages compatible with Conan 2.0. To run this example successfully you should add this
+    remote to your Conan configuration (if did not already do it) doing:
+    ``conan remote add conanv2 https://conanv2beta.jfrog.io/artifactory/api/conan/conan --index 0``
+
+In the :ref:`previous tutorial section<tutorial_creating_packages>` we created a Conan
+package for a "Hello World" C++ library. We used the
+:ref:`conan.tools.scm.Git()<reference>` tool to retrieve the sources from a git
+repository. So far, the package does not have any dependency on other Conan packages.
+Let's explain how to add a dependency to our package in a very similar way that we did in
+the :ref:`consuming packages section<consuming_packages_flexibility_of_conanfile_py>`. We
+will add some fancy colour output to our "Hello World" library using the `fmt 
+<https://conan.io/center/fmt>`__ library.
+
+Please, first clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ on GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/tutorial/creating_packages/add_requires
+
+You will notice some changes in the `conanfile.py` file from the previous recipe.
+Let's check the relevant parts:
+
+.. code-block:: python
+    
+    ...
+    from conan.tools.build import check_min_cppstd
+    ...
+
+    class helloRecipe(ConanFile):
+        name = "hello"
+        version = "1.0"
+
+        ...
+        generators = "CMakeDeps"
+        ...
+
+        def validate(self):
+            check_min_cppstd(self, "11")
+
+        def requirements(self):
+            self.requires("fmt/8.1.1")
+
+        def source(self):
+            git = Git(self)
+            # FIXME: point to conan-io repo before merging
```

Comment:
?

Reply:
Oh! Thanks, yes, I left the FIXME but it was pointing to the correct repo